classtest.js

```diff
@@ -1,87 +1,152 @@
-    timeEl.textContent = cph();
+    timeEl.textContent = storeRandomCPH;
     rowEl.appendChild(timeEl);
-    total += cph();
+    total += storeRandomCPH;
+    this.hourlySales.push(storeRandomCPH);
+    console.log(this.hourlySales);
   }
-  //log to dom the total in a column
-  console.log(currentStore.name + ' ' + total);
+  //---render to dom the total in a column---//
   tableEl.appendChild(rowEl);
   var totalEl = document.createElement('td');
   totalEl.textContent = total;
   rowEl.appendChild(totalEl);
   total = 0;
 };
+var colTotalAdded = document.createElement('tr');
+colTotalAdded.textContent = colTotals;
+tableEl.appendChild(colTotalAdded);
 
-document.body.appendChild(tableEl);
+function columnTotals(){
+  for (var ii = 0; ii < timeSlot.times.length; ii++){
+    var allStoreHourlyTotal = 0;
+    for(var jj = 0; jj < stores.length; jj++){
+      allStoreHourlyTotal += stores[jj].hourlySales[ii];
+    }
+    colTotals.push(allStoreHourlyTotal);
+  };
+  console.log(colTotals);
+}
 
-//------------------------FUNCTIONS----------------------------
-function cph(){
-  var avgCookiesPerHour = Math.ceil((Math.floor(Math.random() * ( currentStore.maxCustomers - currentStore.minCustomers + 1) + currentStore.minCustomers)) * currentStore.avgCookies);
-  return avgCookiesPerHour;
-};
+console.log('------- EVENT LISTENERS--------');
+//---linking the js to the form tag and id on html---//
+var storeFormEl = document.getElementById('new-store-form');
+//---adding an event listener that when submit is executed it runs "handleSubmit" function
+storeFormEl.addEventListener('submit', handleSubmit);
+//---...---//
+function handleSubmit(event){
+  event.preventDefault();//---what does this do??---//
+  event.stopPropagation();//---what does this do??---//
+  var storeName = event.target.cookieStoreName.value;
+  var minCust = parseInt(event.target.minCustomersPerHour.value);
+  var maxCust = parseInt(event.target.maxCustomersPerHour.value);
+  var avgCookies = parseFloat(event.target.avgCookiesPerCustomer.value);
+  // console.log('User Pressed Submit Button on Form');
+  var store = new CookieStore(storeName, minCust, maxCust, avgCookies);
+  stores.push(store);
+  store.renderRow();
+  console.log(stores);
+  console.log(store);
+  console.log('store.getAvgCookieCount: ', store.getAvgCookieCount());
+}
 
-CookieStore.prototype.getAvgCookieCount = function (){
-  var range = this.maxCustomers - this.minCustomers;
-  return Math.floor(Math.random() * range + this.minCustomers);
-};
+//-------------------------VARIABLES-------------------------//
+//---timeslot to loop through for rendering to dom and the length of calculations--//
 
-function CookieStore(name, minCustomers, maxCustomers, avgCookies) {
-  this.name = name;
-  this.minCustomers = minCustomers;
-  this.maxCustomers = maxCustomers;
-  this.avgCookies = avgCookies;
+//------creating a row item that ends up being labled 'times' on the table-------//
+function tableHeader() {
```

Comment:
It might make more sense if your name your functions with verbs, so this could be 'drawTableHeader' instead. The name 'tableHeader' would be appropriate if it were actually holding a table header element.